app/gui/src/dashboard/providers/AuthProvider.tsx

```diff
@@ -693,3 +698,10 @@ export function useFullUserSession(): FullUserSession {
 
   return session
 }
+
+/** A React context hook returning the user session for a user that is fully logged in. */
+export function useUser() {
+  const { user } = useFullUserSession()
+
+  return user
```

Comment:
(comment only, no action needed) this is truly one of the utility functions of all time